platform/kindle/extensions/koreader/bin/koreader-ext.sh

```diff
@@ -115,11 +118,11 @@ set_cre_prop()
 	# Check that the config exists...
 	if [ -f "${cre_config}" ] ; then
 		# dos2unix
-		sed -e "s/$(echo -ne '\r')$//g" -i "${cre_config}"
+		sed -e "s/$(printf '\r')$//g" -i "${cre_config}"
```

Comment:
Yeah, that was okay the way it is and my change was mistaken because I forgot to escape '\r' properly. In this case it's easier to just disable the shellcheck warning (plus then no checking on device is required).